src/components/Editor.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {useState} from 'react';
+
+const Editor = () => {
+
+    //Sets state variable 'changes' where we will be storing text on change
+    const [changes, setChanges] = useState(``);
```

Comment:
Don't use template literals (i.e. the ` symbol) unless you're going to be doing string interpolation or using string templates